sdk/schemaregistry/schema-registry/package.json

```diff
@@ -0,0 +1,124 @@
+{
+  "name": "@azure/schema-registry",
+  "version": "0.1.0",
```

Comment:
I wasn't sure what version to put here.

Reply:
```suggestion
  "version": "1.0.0-preview.1",
```

this is traditional, though we are thinking of changing from `preview` to `prerelease`

Reply:
We can go with `1.0.0-preview.1` for now. This package will take another month to ship, we can change later to prerelease if needed

Reply:
Done.